src/App.tsx

```diff
@@ -1,61 +1,122 @@
+import { ChangeEvent, FormEvent, useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { TodosList } from './types/TodoList';
+
+const combineTodosAndUser = todosFromServer.map(todo => (
+  {
+    ...todo,
+    user: usersFromServer.find(user => user.id === todo.userId),
```

Comment:
find can return undefined, it's better to use null instead of undefined for user